scripts/reindex.py

```diff
@@ -14,12 +14,16 @@
 from urllib import request, error
 import os
 import sys
+from concurrent.futures import ThreadPoolExecutor
+
+from packaging.version import Version
+
 try:
     import conda
 except ImportError:
     conda = None
 
-sys.path.insert(0, str(Path(__file__).parent.parent / 'lib'))
+sys.path.insert(0, str(Path(__file__).parent.parent / "lib"))
```

Comment:
What lib is here used? Is this one from the python environment? If Yes, then I suggest using `sys.prefix` instead of the assumption where the script landed.

Reply:
This line is only changed because I ran "black" on this file after making my changes and that swapped to double-quotes.

Actually I think this line is unnecessary and the line below is the one that is used to allow importing these github utils from `lib.pyapi`. It might be a good idea to better organize these files within the repo at some point. I can do that here if you like but it seems a bit out of scope.

Reply:
Ok. It could be a good idea to add pre-commit to this repository in a separate PR. I will do this later.